src/Login.js

```diff
@@ -1,52 +1,78 @@
-import React from "react";
+import React, {useState} from "react";
+import {login, useAuth} from "./App"
+import {logout} from "./App";
 import {Register} from "./Register";
 import {Route} from "react-router-dom";
+import {createAuthProvider} from 'react-token-auth';
 
-export class Login extends React.Component {
-    constructor(props) {
-        super(props);
 
-        this.state = {
-            login: '',
-            password: ''
-        };
-    }
+function Login (props) {
 
-    onClick = () => {
-        localStorage.setItem('AUTH', "AUTH");
+    const [logdata, setLogdata] = useState({
+        login: '',
+        password: ''
+    });
+    const [errors, setErrors] = useState({});
+    const onClick = () => {
+        const body = {
+            email: logdata.login,
+            password:logdata.password
+        }
+        fetch('/auth/login', {
+            method: 'POST',
+            headers: {
+                'content-type': 'application/json'
+            },
+            body: JSON.stringify(body)
+        })
+            .then(r => {if (r.status!==200){
```

Comment:
I think you need here smth like: 
```
.then(r => {
                if (r.status !== 200) {
                    setErrors({logError: r.message});
                    console.log(body);
                }
                return r.json();
            })
```
In other case `token` in the following `then` is `undefined` and redirect to dashoboard (and saving token) does not happen